packages/notebook-extension/schema/tracker.json

```diff
@@ -377,11 +377,17 @@
       "type": "boolean",
       "default": true
     },
-    "nonObservedBottomMargin": {
-      "title": "Non-observed bottom margin",
-      "description": "Defines the non-observed bottom margin for the virtual notebook, set a positive number of pixels to render cells below the visible view",
+    "observedTopMargin": {
+      "title": "Observed top margin",
+      "description": "Defines the observed top margin for the virtual notebook, set a positive number of pixels to render cells above the visible view",
       "type": "string",
-      "default": "0px"
+      "default": "1000px"
+    },
+    "observedBottomMargin": {
+      "title": "Observed bottom margin",
+      "description": "Defines the observed bottom margin for the virtual notebook, set a positive number of pixels to render cells below the visible view",
+      "type": "string",
+      "default": "1000px"
```

Comment:
Same here. Probably not worth for this PR. Would taking into account zoom levels and have an "auto" option that sets this size taking into account the current screen height, for example?

Reply:
Yep, to be tested for a next tuning PR